lambda/index.py

```diff
@@ -0,0 +1,131 @@
+import re
+import boto3
+import base64
+import subprocess
+import logging
```

Comment:
I really like making 2 distinct changes in a single PR here.

The problem was: the Lambda code is downloaded at synth time. 

The expected solution was: the Lambda code is built at release time and bundled into the NPM package.

You are doing that, and also replacing the code in the Lambda layer with a downloaded binary and a Python wrapper at the same time. 

This seems to me like it doesn't appreciably simplify the solution to the immediate problem, as well as introduces another layer of risk that I'd rather avoid. The proposed layer code change should be submitted as a separate PR.